Octokit.Reactive/Clients/IObservableIssuesClient.cs

```diff
@@ -55,7 +55,7 @@ public interface IObservableIssuesClient
         /// <param name="number">The issue number</param>
         [SuppressMessage("Microsoft.Naming", "CA1716:IdentifiersShouldNotMatchKeywords", MessageId = "Get",
              Justification = "Method makes a network request")]
-        IObservable<Issue> Get(string owner, string name, int number);
```

Comment:
The `number` used in this interface is not the one overflowing. Can we undo the changes in this file to simplify the diff?